### PR TITLE
Fix benchmarking scripts

### PR DESCRIPTION
**Pull request by devernay**
- closes https://github.com/nerfstudio-project/nerfstudio/issues/1000
- see also https://github.com/nerfstudio-project/nerfstudio/issues/806
- launch_train_blender.sh:
  - add -s option to launch a single job per GPU
  - add -v option to use tensorboard instead of wandb
  - set nerfacto options according to https://github.com/nerfstudio-project/nerfstudio/issues/806#issuecomment-1284327844
  - use a single timestamp for all training jobs
  - last GPU was ignored
  - kill all subprocesses when script is terminated
  - print the eval script command-line
-  launch_eval_blender.sh:
    - add shebang
    - add -s option to launch a single job per GPU
    - last GPU was ignored
    - kill all subprocesses when script is terminated
- update benchmarking doc

### CODE REVIEW

File: docs/developer_guides/debugging_tools/benchmarking.md [hunk 1]
```diff
@@ -71,6 +73,7 @@ The flags used in the benchmarking script are defined as follows:
 - `-m`: config name (e.g. `instant-ngp`). This should be the same as what was passed in for -c in the train script.
 - `-o`: base output directory for where all of the benchmarks are stored (e.g. `outputs/`). Corresponds to the `--output-dir` in the base `Config` for training.
 - `-t`: timestamp of benchmark; also the identifier (e.g. `2022-08-10_172517`).
+- `-s`: Launch a single job per GPU.
 - `-g`: specifies the gpus to use and if not specified (no -g flag), will automaticaly search for available gpus.
```
Comment (tancik) [inline docs/developer_guides/debugging_tools/benchmarking.md:77]: Is this flag still used?

Comment (devernay) [reply docs/developer_guides/debugging_tools/benchmarking.md:77]: That's a flag I added. It basically launches one job per GPU, then waits on the first one to be finished before relaunching on that GPU.
In the previous version, all jobs were launched in parallel at script launch (and Ctrl-C didn't kill the jobs). That works fine if you have several GPUs and lots of GPU memory, but not if you have a single 16Gb GPU.

Comment (tancik) [reply docs/developer_guides/debugging_tools/benchmarking.md:77]: Ahh sorry, I was trying to highlight the `-g` flag.

Comment (devernay) [reply docs/developer_guides/debugging_tools/benchmarking.md:77]: well this flag was simply ignored in the previous version of the scripts. It just took the list of remaining arguments as the list of GPUs, so I kept it that way and removed the unused flag. I'll adjust this doc.

**Review by tancik — APPROVED** [review]
LGTM!

**Pull request merged**